src/ApiPort.VisualStudio/Models/OptionsModel.cs

```diff
@@ -61,24 +61,13 @@ public IList<TargetPlatform> Platforms
         public string OutputDirectory
         {
             get { return _outputDirectory; }
-            set
-            {
-                UpdateProperty(ref _outputDirectory, value);
```

Comment:
The default SelectedPath for the dialog is "", so I was wondering if it was possible to press OK and not select a folder on the dialog.